.github/actions/ortdocker/check_image.py

```diff
@@ -26,7 +27,11 @@
 token = os.getenv("INPUT_TOKEN")
 org = os.getenv("GITHUB_REPOSITORY_OWNER")
 name = os.getenv("INPUT_NAME")
-version = os.getenv("INPUT_VERSION")
+base_version = os.getenv("INPUT_VERSION")
+unique_id = hashlib.sha256(os.getenv("BUILD_ARGS").encode()).hexdigest()
```

Comment:
Is this hashing supposed to solve https://github.com/oss-review-toolkit/ort/issues/7798 ?

How does it work with changes which do not affect build args? 
e.g. I've added `git-lfs` a while ago, but in the published image it's not available for unknown reason. If I build image locally it works as expected.

Reply:
Yes, this supposes to solve this specific bug.
When build args not changes, hash will be the same, so will check if the image tagged in ghcr.io with {version}-{hash} exists. If exists, not build, since is the same.

Reply:
I don't understand yet why 'git-lfs' is not in the docker built by CI, see [1].
That commit [1] is not changing build args, but I wonder if your change here fixes that problem implicitly, e.g. so that docker layer caching starts detecting the modified `apt-get install` e.g.

Can you help me understanding this bit?

[1] https://github.com/oss-review-toolkit/ort/commit/a4b12df0acc6329cc164712e189d2c865c4280b2

Reply:
Images suppose to be immutable, .
The check happens in this way:
current: base-17-jdk-jammy,  so the version is 17-jdk-jammy.

The test before was checking for version, so every time found a container in ghcr with the version that matches, simply pass and not build it. 

The problem is that we changed the internal content without change the main version. 
So when you added git-lfs, even changing inside, the new container was never generated.

Noe we have a hash based on the arguments as part of the version, so change the build arguments will declare as a new image and will build and push as new.

And the main registry will not receive pushes from content during pull_requests. Only in nighlty or in the tag events.
After this PR is approved, we trigger a workflow dispatch event and the issue will be gone,

Reply:
Thanks!